RELEASE_NOTES.md

```diff
@@ -1,7 +1,16 @@
 # Cosmos SDK v0.42.0 "Stargate" Release Notes
 
-This release contains a single, but important security fix for all non "Cosmos Hub" chains (e.g. any chain that does not use the default `cosmos` bech32 prefix). The fix addresses a bug in evidence handling on the Cosmos SDK that rendered the `v0.41.x` and `v0.40.x` release series unsafe for most chains. Please see the PR below for more details.
+This release includes an important security fix for all non "Cosmos Hub" chains (e.g. any chain that does not use the default `cosmos` bech32 prefix), and a few performance improvements.
 
-## Bug Fixes
+See the [Cosmos SDK v0.42.0 milestone](https://github.com/cosmos/cosmos-sdk/milestone/42?closed=1) on our issue tracker for further details.
 
-- [#8461](https://github.com/cosmos/cosmos-sdk/pull/8461) Fix bech32 prefix in evidence validator address conversion
+# Security fix: validator address conversion in evidence handling
+
+The security fix resolves the issue regarding incorrect handling of validators' consensus addresses. Because of this incorrect handling, Cosmos SDK apps that were not using the default `cosmos1` Bech32 address prefix were not able to jail validators that committed misbehaviors such as double signing.
```

Comment:
```suggestion
The security fix resolves the issue regarding incorrect handling of validators' consensus addresses. Because of this incorrect handling, Cosmos SDK apps that were not using the default `cosmos` Bech32 address prefix were not able to jail validators that committed misbehaviors such as double signing.
```